sdk/search/search-documents/src/generated/data/models/index.ts

```diff
@@ -437,20 +445,20 @@ export interface AutocompleteOptions {
   top?: number;
 }
 
-/** Known values of {@link ApiVersion20200630Preview} that the service accepts. */
-export const enum KnownApiVersion20200630Preview {
-  /** Api Version '2020-06-30-Preview' */
-  TwoThousandTwenty0630Preview = "2020-06-30-Preview"
+/** Known values of {@link ApiVersion20210430Preview} that the service accepts. */
+export const enum KnownApiVersion20210430Preview {
```

Comment:
what's the point in this enum? It only has a single value (rather than listing all known/supported api versions) and from its name it doesn't seem like it would ever contain other values